code/modules/antagonists/nukeop/equipment/borgchameleon.dm

```diff
@@ -11,8 +11,8 @@
 	var/friendlyName
 	var/savedName
 	var/active = FALSE
-	var/activationCost = 300
-	var/activationUpkeep = 50
+	var/activationCost = 300 KILO JOULES
+	var/activationUpkeep = 50 KILO JOULES
```

Comment:
You could convert the activation upkeep to watts and scale the cell usage by seconds_per_tick. Cleaner that way.